Flaky UI test in CI

We appear to have a flaky test in the UI. This worked after re-running the job.

https://github.com/mergestat/mergestat/actions/runs/3609487739/jobs/6082710890

```      
      116 |     )
      117 |
    > 118 |     await waitFor(() => {
          |                  ^
      119 |       // Get input search to type
      120 |       const searchInput = screen.getByTestId(TEST_IDS.inputRepoSearch)
      121 |       if (searchInput) {

      at waitForWrapper (node_modules/@testing-library/dom/dist/wait-for.js:187:27)
      at Object.<anonymous> (src/__tests__/repos.test.tsx:118:18)

PASS src/__tests__/syncs.test.tsx (6.369 s)
PASS src/__tests__/settings.test.tsx

Test Suites: 1 failed, 3 passed, 4 total
Tests:       1 failed, 22 passed, 23 total
Snapshots:   0 total
Time:        20.169 s
Error: Process completed with exit code 1.
```

--- a/ui/jest.config.js
+++ b/ui/jest.config.js
@@ -2,6 +2,7 @@ const nextJest = require('next/jest')
 
 const createJestConfig = nextJest({
   dir: './',
+  testTimeout: 20000,
 })
 
 const customJestConfig = {

